common/changes/office-ui-fabric-react/documentcard-basic-example-href-change_2019-01-14-18-37.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Changed href in basic example to bing.com",
```

Instead maybe suggest a change for the comment itself 😄 
```suggestion
      "comment": "DocumentCard: changed href in basic example to bing.com",
```